### PR TITLE
Travis and tests (Closes #5)

### LINKED ISSUE #5: The project is missing CI
This project should have a CI, probably Travis

### PR DESCRIPTION
**Pull request by gabrielerzinger**
Adds travis ci and also solve a racing condition where pitaya admin was not ready for tests. I believe that the sleep at package testing is not ideal and after the CI, Im going to improve that

Also, fixed the request test to match the logic we used in docs test - not checking for the actual answer and, instead, checking if it is coherent with what is expected

### CODE REVIEW

**Review by cscatolini — APPROVED** [review]
Great job :)

Travis is running as expected: https://travis-ci.org/topfreegames/pitaya-admin. 

Please include the travis badge in the readme and then I'll merge this,

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: go
+go:
+  - "1.10"
```
Comment (andrehp) [inline .travis.yml]: Use 1.11, it's the newest version

**Review by andrehp — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,222 +1,234 @@
...
+[2]: https://travis-ci.org/topfreegames/pitaya-admin
+[3]: https://godoc.org/github.com/topfreegames/pitaya-admin?status.svg
+[4]: https://godoc.org/github.com/topfreegames/pitaya-admin
+[5]: https://goreportcard.com/badge/github.com/topfreegames/pitaya-admin
+[6]: https://goreportcard.com/report/github.com/topfreegames/pitaya-admin
+[7]: https://img.shields.io/badge/license-MIT-blue.svg
+[8]: LICENSE
 
 [Pitaya's](https://github.com/topfreegames/pitaya) games management and monitoring interface
+sdasd
```
Comment (cscatolini) [inline README.md]: 🤔

Comment (gabrielerzinger) [reply README.md]: 🤦‍♂️

**Pull request merged**